atom/browser/api/atom_api_cookies.cc

```diff
@@ -159,7 +159,8 @@ void RunCallbackInUI(base::OnceClosure callback) {
 // Remove cookies from |list| not matching |filter|, and pass it to |callback|.
 void FilterCookies(std::unique_ptr<base::DictionaryValue> filter,
                    util::Promise promise,
-                   const net::CookieList& list) {
+                   const net::CookieList& list,
+                   const net::CookieStatusList& excluded_list) {
```

Comment:
It includes the cookies ignored while using the `SetCanonicalCookieAsync` and is populated when using the cookie getter apis, but since we provide the error code when setting the cookie its not necessary for us to return this list. Its useful for browsers to display the excluded cookies during a requeset in devtools panel.